homeassistant/components/homee/__init__.py

```diff
@@ -0,0 +1,274 @@
+"""The homee integration."""
+
+from dataclasses import dataclass
+import logging
+from typing import Any
+
+from pyHomee import Homee
+from pyHomee.const import AttributeType, NodeProfile
+from pyHomee.model import HomeeAttribute, HomeeNode
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.typing import ConfigType
+
+from .const import ATTR_HOMEE_DATA, CONF_ADD_HOMEE_DATA, DOMAIN
+from .helpers import get_name_for_enum
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
No need to have a YAML config here

Reply:
Now I get on Commit:
Integration homee:
* [ERROR] [CONFIG_SCHEMA] Integrations which implement 'async_setup' or 'setup' must define either 'CONFIG_SCHEMA', 'PLATFORM_SCHEMA' or 'PLATFORM_SCHEMA_BASE'. If the integration has no configuration parameters, can only be set up from platforms or can only be set up from config entries, one of the helpers cv.empty_config_schema, cv.platform_only_config_schema or cv.config_entry_only_config_schema can be used.

Reply:
But you don't need async setup

Reply:
Ah, then I can cope with that, when I add services...

Reply:
Exactly!